app/src/main/java/org/mifos/mobile/ui/savings_account_application/SavingsAccountApplicationFragment.kt

```diff
@@ -0,0 +1,145 @@
+package org.mifos.mobile.ui.savings_account_application
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Toast
+import androidx.compose.ui.platform.ComposeView
+import androidx.compose.ui.platform.ViewCompositionStrategy
+import androidx.fragment.app.viewModels
+import dagger.hilt.android.AndroidEntryPoint
+import org.mifos.mobile.R
+import org.mifos.mobile.api.local.PreferencesHelper
+import org.mifos.mobile.core.ui.theme.MifosMobileTheme
+import org.mifos.mobile.models.accounts.savings.SavingsAccountApplicationPayload
+import org.mifos.mobile.models.accounts.savings.SavingsAccountUpdatePayload
+import org.mifos.mobile.models.accounts.savings.SavingsWithAssociations
+import org.mifos.mobile.ui.activities.base.BaseActivity
+import org.mifos.mobile.ui.enums.SavingsAccountState
+import org.mifos.mobile.ui.fragments.base.BaseFragment
+import org.mifos.mobile.utils.Constants
+import org.mifos.mobile.utils.DateHelper
+import org.mifos.mobile.utils.Network
+import org.mifos.mobile.utils.ParcelableAndSerializableUtils.getCheckedParcelable
+import org.mifos.mobile.utils.ParcelableAndSerializableUtils.getCheckedSerializable
+import org.mifos.mobile.utils.getTodayFormatted
+import javax.inject.Inject
+
+/*
+* Created by saksham on 30/June/2018
+*/
+@AndroidEntryPoint
+class SavingsAccountApplicationFragment : BaseFragment() {
+
+    @Inject
+    lateinit var preferencesHelper: PreferencesHelper
```

Comment:
This also comes to data computations and It should be in ViewModel, move it there.